package.json

```diff
@@ -25,7 +25,7 @@
     "@0xpolygonhermez/zkevm-commonjs": "github:0xpolygonhermez/zkevm-commonjs#v1.0.0",
     "@axelar-network/axelar-cgp-solidity": "6.2.1",
     "@axelar-network/axelar-gmp-sdk-solidity": "5.6.4",
-    "@axelar-network/interchain-token-service": "1.2.1",
+    "@axelar-network/interchain-token-service": "https://github.com/axelarnetwork/interchain-token-service.git",
```

Comment:
We already discussed this, but once we have a stable release changing it here should fix the tests